config/jobs/kubernetes/sig-release/release-branch-jobs/1.12.yaml

```diff
@@ -600,3 +628,72 @@ periodics:
           resources:
             requests:
               cpu: 4
+  - name: ci-kubernetes-e2e-gce-device-plugin-gpu-stable2
+    cron: "0 8-23/12 * * *"
+    labels:
+      preset-service-account: "true"
+      preset-k8s-ssh: "true"
+      preset-ci-gce-device-plugin-gpu: "true"
+    annotations:
+      testgrid-dashboards: sig-release-1.12-blocking, sig-release-1.12-all
+      testgrid-tab-name: gce-device-gpu-plugin-1.12
+    spec:
+      containers:
+        - args:
+            - --timeout=300
+            - --bare
+            - --scenario=kubernetes_e2e
+            - --
+            - --check-leaked-resources
+            - --extract=ci/k8s-stable2
+            - --gcp-project-type=gpu-project
+            - --gcp-zone=us-west1-b
+            - --provider=gce
+            - --test_args=--ginkgo.focus=\[Feature:GPUDevicePlugin\] --minStartupPods=8
+            - --timeout=180m
+          image: gcr.io/k8s-testimages/kubekins-e2e:v20190509-e418529-master
+  # This is a sig-release-<stable2>-blocking job.
```

Comment:
(1) Can we use the description annotation instead of the comments?

Reply:
These jobs all previously existed and are just cut/paste. Improving the jobs is out of scope for this PR.